src/app/client/src/app/modules/core/components/main-header/main-header.component.html

```diff
@@ -404,6 +404,14 @@ <h4 class="mb-4 text-capitalize">{{showingResult}}</h4>
               </svg></i>
             {{resourceService.frmelmnts?.lnk?.dashboard}}
           </li>
+          <li class="list-item" *ngIf='!isDesktopApp && permissionService.permissionAvailable' appPermission
+          [permission]='programDashboardRole' (click)="toggleSideMenu(false)" routerLink="program">
+          <i class="icon-svg icon-svg--sm icon-dashboard mr-8"><svg class="icon icon-svg--primary">
+              <use xlink:href="./assets/images/sprite.svg#dashboard"></use>
+            </svg></i>
+            Program Dashboard
```

Comment:
please do not hard code the value

Reply:
for this and the dashboard the image is the same?

Reply:
I have fixed hard code value, Image is same only, program-dashboard can be accessible only with role of PROGRAM_MANAGER and PROGRAM_DESIGNER